cards.js

```diff
@@ -16,8 +28,47 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
-
-// Put some code in here!
+var deck = []
+function buildDeck(suits, values){
+  for(var index_v in values){
+    for(var index_s in suits){
+      var card = {suit: suits[index_s], value: values[index_v]};
+      deck.push(card);
+    }
+  }
+  return deck
+}
 
+// split deck between two players
+var player1 = []
+var player2 = []
+function dealCards(deck){
+  for(var i=0; i < deck.length; i++){
+    if (i % 2 === 0 ){
+      player1.push(deck[i]);
+    }
+    else{
+      player2.push(deck[i]);
+    }
+  }
+  return deck
 }
+
+//confirm cards are dealt cards out to players
+deck = buildDeck(suits, values)
+shuffledDeck = shuffle(deck)
+dealCards(shuffledDeck)
+console.log("Player 1:  ")
+console.log(player1)
+console.log("Player 2:  ")
+console.log(player2)
+
+
+// // compare cards from two players
+// function highCard(cards){
+//   for(var card in cards){
+//
+//   }
+// }
+
+// Print out to the console the two cards, and which player had the higher card (or if it was a tie).
```

Comment:
i think you're really close to a solution here, when you find more time, i encourage you to finish your solution for high card. I do like that you psuedo coded so you know exactly how you can code this going forward. :+1: good job.